webpack/assets/javascripts/react_app/redux/API/APIRequest.js

```diff
@@ -25,6 +26,9 @@ export const apiRequest = async (
 ) => {
   const { REQUEST, SUCCESS, FAILURE } = actionTypeGenerator(key, actionTypes);
   const modifiedPayload = { ...payload, url };
+  const stopIntervalCallback = selectDoesIntervalExist(getState(), key)
+    ? () => dispatch(stopInterval(key))
+    : () => console.warn(`There's no interval API request for the key: ${key}`);
```

Comment:
is this warning needed in production?

Reply:
Yes I think so, it will be easier for debugging